src/main/java/mate.academy/Main.java

```diff
@@ -6,9 +6,7 @@
 import mate.academy.service.ProductService;
 
 public class Main {
-
     public static void main(String[] args) {
-        // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
         ProductService productService = null;
```

Comment:
Please test your Injector here. Feel free to push this class as a part of your solution